app/src/fetcher.tsx

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+import { config } from './axios-config';
+
+export const getAllData = async () => {
+  const { data } = await axios.get(
+    'http://127.0.0.1:8000/api/tasks',
+    config
+  )
+  return data;
+};
```

Comment:
`axios.get` に型引数でレスポンス型を渡すと、内部で annotation を済ませてくれると思います。
```suggestion
export const getAllData = async () => {
  const { data } = await axios.get<{ tasks: Task[] }>(
    'http://127.0.0.1:8000/api/tasks',
    config
  )
  return data;
};
```

Reply:
そうなんですね！無理やりmapしないといけないかと思ってました！修正します！